examples/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
We use Apache 2.0 licensing. Refer to our existing package.json files for existing attributes.